app/controllers/state_file/questions/az_eligibility_out_of_state_income_controller.rb

```diff
@@ -2,6 +2,8 @@ module StateFile
   module Questions
     class AzEligibilityOutOfStateIncomeController < QuestionsController
       include EligibilityOffboardingConcern
+
+      before_action -> { @filing_year = Rails.configuration.statefile_current_tax_year }
```

Comment:
so admittedly i forgot about this until part way into reviewing this PR but if you go look at `StateFileControllerConcern` there is a helper method called `current_tax_year` which actually gives you access to this config variable via the `MultiTenantService`. so i think you should just remove all of these new before_actions and use that helper method in the views instead